client/components/community-translator/utils.js

```diff
@@ -0,0 +1,177 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import request from 'superagent';
+import { head, find, get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import userSettings from 'lib/user-settings';
+import { isMobile } from 'lib/viewport';
+import { GP_PROJECT, GP_BASE_URL, GP_PROJECT_TRANSLATION_SET_SLUGS } from './constants';
+import { hasTranslationSet } from 'lib/i18n-utils';
+
+/**
+ * Checks whether the CT can be displayed, that is, if the chosen locale and device allow it
+ * @param {String} locale user's localeSlug
+ * @param {Object} localeVariant user's localeVariant slug (if any)
+ * @returns {Boolean} whether the CT can be displayed
+ */
+export function canDisplayCommunityTranslator(
+	locale = userSettings.getSetting( 'language' ),
+	localeVariant = userSettings.getSetting( 'locale_variant' )
+) {
+	// restrict mobile devices from translator for now while we refine touch interactions
+	if ( isMobile() ) {
+		return false;
+	}
+
+	// disable for locales
```

Comment:
Just noticed that this comment is ambiguous. Could we elaborate more details on this?
My best guess is that it is saying "disable for locales without official GP translation sets.

Reply:
You're guess is right. Thanks for picking up on that detail!!